pkg/service/redisstore.go

```diff
@@ -625,6 +627,11 @@ func (s *RedisStore) storeIngressState(_ context.Context, ingressId string, stat
 				return ingress.ErrIngressOutOfDate
 			}
 
+			if state.StartedAt == oldStartedAt && state.UpdatedAt < oldUpdatedAt {
```

Comment:
Mmm, should this `if` contain `return nil` or something similar?

Reply:
Yes, I typoed that line out when I added the comment I guess. Thanks for catching it.